### ISSUE TITLE
Missing alt tag in the context of aria-label

### ISSUE THREAD

**Issue by sethshober** [GoogleChrome/lighthouse#6861]
I'm not sure if this is actually a bug or feature request. It's in regards to how `alt` tags are judged. I think we can agree that not including them is a serious hit on accessibility, but my question is in regards to the context. For instance I saw a site get hit by about 10 percent for not having an alt tag on an `img`, where the `img` was nested within a `button`, and the `button` already had a label.

Note: The `img` was just a sprite sheet showing a `+` sign. Isn't this label on the button equivalent in this situation, in regards to accessibility? Should there be such a hit on the site rating when missing an `alt` tag, given this context?

Example:
```
<button title="Zoom in" aria-label="Zoom in" type="button">
  <div>
    <img alt src="maps.google.com">
  </div>
</button>
```

**Comment by Hoten** [comment]
> [Having] a "null" alt attribute is not the same as having no alt attribute.
[Source](https://www.w3.org/TR/WCAG20-TECHS/H67.html#H67-description)

It's simpler and safer to enforce an `alt` attribute on everything, even a blank one to signify it's un-importance.

**Comment by sethshober** [comment]
Gotcha. Thanks for the clarification. I didn't realize any empty one would be interpreted differently. I will suggest to update with empty alt values rather than none.
